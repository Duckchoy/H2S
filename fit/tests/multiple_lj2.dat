0 0.000175813389875      -0.240403998522525      0.004059737214175      -1.240320306123500      -19469.0616179229      0.238686602138741
0 0.000092182491025      -0.183442153938825      0.001734021272225      -0.879631675452150      -19472.9120892096      0.182401643159099
0 0.000049860284075      -0.142181075095025      0.000789655381475      -0.640647235786800      -19473.4094089342      0.182166503514693
0 0.000015916676050      -0.089042888161775      0.000192134472075      -0.362916405310925      -19473.6622826838      0.154010797385904
0 0.000009362139950      -0.071771090191525      0.000101348203250      -0.280735363729050      -19473.2305595504      0.177090111454995
0 0.000001752986425      -0.036506867970250      0.000014190657850      -0.127642768154075      -19471.3645007981      0.246164621207460
0 0.000175813389875      -0.240403998522525      0.000060318030875      -0.227977760441375      -19473.7440847660      0.183454670180187
0 0.000092182491025      -0.183442153938825      0.000034024986075      -0.181217285312200      -19473.5097301091      0.181134859003763
0 0.000049860284075      -0.142181075095025      0.000019791865975      -0.145844729991500      -19473.0626142083      0.157694766608642
0 0.000015916676050      -0.089042888161775      0.000007256638850      -0.097566945224425      -19472.3065436056      0.157256413877736
0 0.000009362139950      -0.071771090191525      0.000004551507000      -0.080936640643075      -19471.4234401913      0.127487392760343
0 0.000001752986425      -0.036506867970250      0.000001035711450      -0.044722439537450      -19472.2609793814      0.161468108687617
0 0.000000613112900      -0.023939936568850      0.000000405524625      -0.030707137126825      -19471.3614887622      0.171103862579763
0 0.000000099160150      -0.011523439716550      0.000000078128450      -0.015850882642400      -19471.8054762213      0.143659329830843
1 0.000031553731500      -0.117275528554825      0.000776266855325      -0.539446909417150      -19487.3424334315      0.221558464915048
1 0.000000021229225      -0.006201510360075      0.000000109046700      -0.017175706809200      -19486.7965894514      0.140129703309288

