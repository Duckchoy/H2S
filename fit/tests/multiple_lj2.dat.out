prob -192.2278165

 new best -19471.83338 -19486.80429 390.1934541 8.765131242 256.6060779 -0.7437443868 -192.2278165
prob -338.9192258
prob -193.100552
prob -279.6699441
prob -142.2287023

 new best -19471.75138 -19486.73562 -111.1867914 7.881433557 693.1131344 0.4262532579 -142.2287023
prob -204.3142705
prob -307.5575388
prob -185.4991793
prob -312.6648666
prob -202.3585043
prob -327.6675175
prob -260.001096
prob -254.9949209
prob -275.6170331
prob -171.1789238
prob -203.781514
prob -148.1333073
prob -300.4451812
prob -309.0245145
prob -363.2752867
prob -165.9246644
prob -64.92379328

 new best -19471.56627 -19486.56986 -571.002798 5.960147925 1691.391765 3.16203035 -64.92379328
prob -194.9058458
prob -160.2900888
prob -348.0107141
prob -289.1917704
prob -234.5850299
prob -283.3911537
prob -179.403956
prob -214.8489538
prob -268.6621924
prob -208.9227073
prob -86.2851197
prob -214.2230809
prob -282.1868587
prob -291.505151
prob -287.0632511
prob -218.1124901
prob -328.6638754
prob -307.4342062
prob -254.218553
prob -299.7415304
prob -120.2616594
prob -254.2424975
prob -231.6056095
prob -188.4922978
prob -311.8105816
prob -200.0299063
prob -299.0609249
prob -162.1474312
prob -247.0132095
prob -141.2187108
prob -191.1453971
prob -198.9832631
prob -216.3587061
prob -294.5741959
prob -322.5435441
prob -274.1952836
prob -195.1074009
prob -342.3469873
prob -228.7085943
prob -213.6517678
prob -111.2386577
prob -152.5876082
prob -157.0992164
prob -267.9901028
prob -302.1747813
prob -145.7698739
prob -135.7018023
prob -126.9429127
prob -308.1781333
prob -270.4885418
prob -250.7401259
prob -232.5557666
prob -243.6549997
prob -201.9451701
prob -418.1388881
prob -236.2388973
prob -330.0553997
prob -147.7989627
prob -252.3377966
prob -215.0739763
prob -204.7637215
prob -161.0196294
prob -216.9188162
prob -321.2049077
prob -348.6348176
prob -131.6966325
prob -247.7460856
prob -254.9075623
prob -215.8594753
prob -156.2769119
prob -290.2341384
prob -220.6393519
prob -313.2893992
prob -271.6351939
prob -202.8662757
prob -98.68291693
prob -195.4853022
prob -205.739501
prob -174.0142138
prob -192.6473272
prob -283.4632239
prob -340.0375946
prob -223.0651916
prob -278.9531279
prob -236.0422912
prob -191.0361816
prob -319.2315064
prob -251.4174958
prob -155.8510022
prob -382.184386
prob -193.6443638
prob -256.304556
prob -308.3895548
prob -274.4496477
prob -176.2737724
prob -273.3278964
prob -267.3650236
prob -275.7327742
prob -327.5503119
prob -182.9060544
prob -196.9217263
prob -254.7336484
prob -268.9217139
prob -331.4404746
prob -390.0311465
prob -221.2134675
prob -266.3851433
prob -183.0283104
prob -224.0564821
prob -307.1116382
prob -172.0675266
prob -203.9333446
prob -110.967606
prob -262.3457431
prob -218.7164368
prob -231.8451654
prob -263.0730957
prob -239.0728089
prob -176.4832746
prob -295.180769
prob -258.0317844
prob -211.1968059
prob -206.5022009
prob -164.5193926
prob -404.2475393
prob -166.1014502
prob -183.8516278
prob -243.2485952
prob -157.3336817
prob -264.5781407
prob -290.38413
prob -129.7970975
prob -164.4045841
prob -154.0801779
prob -154.8430069
prob -116.5640873
prob -473.2706428
prob -328.5872524
prob -229.9926696
prob -410.3344239
prob -201.8191084
prob -110.0457906
prob -197.9107135
prob -285.2689998
prob -273.6542524
prob -271.3300237
prob -250.3187494
prob -272.0199161
prob -109.4158679
prob -314.2037546
prob -262.3162558
prob -148.827767
prob -256.5992849
prob -314.3658691
prob -114.626019
prob -170.9952391
prob -392.8027344
prob -299.2610727
prob -135.757845
prob -223.679509
prob -265.5639333
prob -193.3157683
prob -219.6995813
prob -271.1114478
prob -244.4148457
prob -202.0853407
prob -263.2049526
prob -293.397947
prob -159.4874141
prob -195.3173165
prob -130.3635342
prob -296.3846062
prob -211.0800332
prob -309.3723249
prob -221.6021746
prob -316.198947
prob -318.7445856
prob -295.3562569
prob -223.1617493
prob -443.2728547
prob -286.2441255
prob -277.9382989
prob -285.5422259
prob -361.6723567
prob -130.4999187
prob -240.3938024
prob -180.4629529
prob -205.1602539
prob -164.8020789
prob -248.2215575
prob -171.1918397
prob -171.6192443
prob -262.0555568
prob -569.7791004
prob -293.6519377
prob -223.4822474
prob -321.8801456
prob -197.0466441
prob -223.7710297
prob -292.4473324
prob -230.8552173
prob -215.0049305
prob -167.4017265
prob -117.979336
prob -105.4360401
prob -279.7923437
prob -206.4851843
prob -171.3117154
prob -180.5504012
prob -240.6680177
prob -188.9708949
prob -155.4841429
prob -355.7858265
prob -205.5066454
prob -202.8075673
prob -410.0957724
prob -260.3019578
prob -379.690653
prob -231.2392842
prob -180.9347435
prob -350.8050278
prob -363.8044825
prob -167.5096546
prob -351.3000531
prob -242.2350043
prob -166.1642541
prob -163.7869465
prob -136.1567996
prob -393.8839601
prob -168.9414118
prob -276.548056
prob -242.293712
prob -329.5013479
prob -263.0179518
prob -259.6151866
prob -152.4922211
prob -224.8488454
prob -230.1699609
prob -190.5629794
prob -219.6258024
prob -208.4517769
prob -220.3848542
prob -146.4899846
prob -286.1022906
prob -238.3345461
prob -218.5507051
prob -203.5305793
prob -151.1150375
prob -278.8131769
prob -154.0172425
prob -147.7879112
prob -395.1576467
prob -135.8930712
prob -163.0723371
prob -221.8895628
prob -226.4833613
prob -187.512644
prob -264.0725674
prob -236.8528709
prob -178.7448219
prob -294.1478205
prob -151.3739603
prob -272.8162718
prob -113.5632845
prob -229.8033111
prob -185.5099156
prob -220.353548
prob -292.274709
prob -208.5649295
prob -243.1534067
prob -280.6698722
prob -207.9070302
prob -232.2212843
prob -255.0395803
prob -157.3668769
prob -280.8711611
prob -243.6629345
prob -267.5000002
prob -258.3879135
prob -168.6721391
prob -194.6025792
prob -358.9987473
prob -197.8154358
prob -184.2882399
prob -128.0859922
prob -324.7234103
prob -237.8453517
prob -417.0326617
prob -199.6742451
prob -202.8277304
prob -156.8583102
prob -175.0272288
prob -260.6047347
prob -252.6138487
prob -226.4582905
prob -267.5010254
prob -361.8547121
prob -250.6061327
prob -163.7652197
prob -334.4210712
prob -281.814951
prob -250.0450952
prob -207.2512838
prob -367.7123867
prob -384.4848702
prob -279.0687999
prob -289.3172384
prob -276.7663982
prob -363.4517848
prob -206.7067147
prob -206.1815688
prob -344.9909859
prob -277.5072335
prob -479.1118313
prob -122.9625122
prob -167.2896866
prob -226.2812742
prob -134.2445078
prob -198.5621743
prob -177.3274399
prob -178.6534255
prob -302.3469107
prob -210.2879698
prob -161.2808228
prob -270.1904733
prob -290.0545198
prob -299.7945198
prob -241.4307566
prob -222.8933911
prob -205.3754064
prob -271.0944303
prob -263.7780581
prob -169.2485859
prob -195.5912964
prob -396.7675297
prob -234.1525017
prob -318.4074975
prob -215.5550464
prob -294.4832511
prob -219.8939335
prob -376.2355915
prob -199.6905575
prob -246.7139346
prob -239.5485447
prob -204.0570295
prob -94.14893207
prob -257.6748695
prob -414.8494638
prob -317.552109
prob -292.1790915
prob -206.5665386
prob -143.2131089
prob -258.9733565
prob -109.6071404
prob -303.2030382
prob -91.0243723
prob -119.5992163
prob -145.2029635
prob -246.0872682
prob -227.1667244
prob -285.8648289
prob -169.3909204
prob -371.6505012
prob -172.2264892
prob -203.9173904
prob -144.997819
prob -219.3769617
prob -178.1506476
prob -167.0850671
prob -223.9426476
prob -367.6171626
prob -170.9168468
prob -288.3095249
prob -214.8037108
prob -266.2812592
prob -306.2109965
prob -185.6623362
prob -360.9069267
prob -297.1417006
prob -200.9494449
prob -301.8952706
prob -239.566812
prob -237.3428566
prob -122.5004173
prob -226.4881971
prob -270.1083112
prob -218.3091808
prob -132.9850113
prob -268.789988
prob -220.6826489
prob -269.8381251
prob -200.9610326
prob -118.947683
prob -200.773135
prob -436.2484769
prob -130.1301777
prob -221.774702
prob -110.2029932
prob -209.0157114
prob -101.3965913
prob -331.5620564
prob -196.3718836
prob -309.898655
prob -182.8352877
prob -272.6644359
prob -154.6959167
prob -258.5033819
prob -202.4314026
prob -183.9099006
prob -236.0483545
prob -267.2878362
prob -123.6218325
prob -285.4482072
prob -204.9314397
prob -128.0900107
prob -346.2106292
prob -188.6270187
prob -198.1451348
prob -207.9460465
prob -246.7904557
prob -364.7590783
prob -263.8890481
prob -255.8987332
prob -273.5155899
prob -165.7141048
prob -354.4013548
prob -231.2180251
prob -180.9349649
prob -205.0612898
prob -313.8621459
prob -344.4685038
prob -215.827839
prob -305.0252974
prob -292.9188393
prob -90.87629148
prob -195.17548
prob -244.521485
prob -438.2225281
prob -170.2918546
prob -327.8285115
prob -178.6406647
prob -233.4204309
prob -198.0147289
prob -190.0453563
prob -296.1341078
prob -141.3192259
prob -406.6440728
prob -206.9548735
prob -303.0010436
prob -195.7280447
prob -345.6878099
prob -248.8747682
prob -261.4038614
prob -233.2898644
prob -386.2392519
prob -249.0497935
prob -304.5765482
prob -313.6160171
prob -145.3633029
prob -311.935021
prob -244.6850532
prob -251.2497651
prob -133.4174617
prob -286.3167963
prob -298.7058544
prob -112.1991447
prob -231.4647363
prob -120.4813265
prob -425.2735574
prob -194.3153614
prob -138.7299647
prob -305.2177715
prob -174.7211725
prob -201.7625646
prob -411.4245789
prob -270.2702922
prob -204.5476282
prob -236.4532535
prob -112.4344546
prob -135.5126863
prob -226.772426
prob -301.8445844
prob -351.4200376
prob -166.9439613
prob -231.1459543
prob -191.9137277
prob -236.3553079
prob -187.1192205
prob -246.613615
prob -302.7916525
prob -228.6973887
prob -227.9000952
prob -257.1708147
prob -144.6443714
prob -215.8874752
prob -290.1088813
prob -211.8555992
prob -225.6951706
prob -204.4907041
prob -112.9836802
prob -373.7329113
prob -278.1696586
prob -167.2175284
prob -298.8310239
prob -247.8068142
prob -220.0338599
prob -168.604665
prob -339.3048848
prob -277.0435006
prob -231.8614372
prob -265.7344169
prob -212.0781766
prob -144.297315
prob -179.1339352
prob -243.7717742
prob -371.7486597
prob -217.3697273
prob -163.1871652
prob -214.9628477
prob -232.0232328
prob -206.6519505
prob -291.1769837
prob -283.9878901
prob -158.9542255
prob -272.2371293
prob -313.6664604
prob -391.5461047
prob -245.0524693
prob -245.2009109
prob -340.7951781
prob -242.4041495
prob -216.7331922
prob -268.3066568
prob -160.9779065
prob -233.3952179
prob -173.8347679
prob -162.881448
prob -186.2079068
prob -344.4706214
prob -253.4888933
prob -123.8458151
prob -197.8784446
prob -202.3566089
prob -196.4678842
prob -150.0310622
prob -203.6928364
prob -197.5237285
prob -216.2373576
prob -169.463751
prob -228.8043613
prob -256.2374863
prob -293.4629179
prob -238.1815234
prob -252.8760277
prob -147.7493253
prob -160.9840063
prob -353.6901371
prob -318.1358151
prob -228.2959986
prob -164.2580619
prob -225.6765546
prob -300.7137883
prob -188.046131
prob -221.5844174
prob -373.5996066
prob -168.9755537
prob -171.1399336
prob -115.8003136
prob -146.570429
prob -247.9667166
prob -266.182342
prob -489.1439057
prob -310.249237
prob -149.9610406
prob -333.4852764
prob -266.7192739
prob -182.2169804
prob -490.782578
prob -272.1201447
prob -188.0584828
prob -175.9982355
prob -259.5610975
prob -211.4348567
prob -275.9636982
prob -153.4828029
prob -176.4726114
prob -218.9346628
prob -327.8703103
prob -244.0289883
prob -337.4303354
prob -298.9810502
prob -218.1045425
prob -243.5845859
prob -213.8283835
prob -310.6111187
prob -193.9216904
prob -288.2414787
prob -230.8422727
prob -258.5894845
prob -209.4017317
prob -211.8127955
prob -152.3142649
prob -102.6266546
prob -333.9687321
prob -188.3999748
prob -165.1092438
prob -282.6159041
prob -188.5071514
prob -255.3562737
prob -392.9971986
prob -221.3347936
prob -264.5899146
prob -400.7400262
prob -173.9764389
prob -211.8506378
prob -136.1528735
prob -296.8985778
prob -350.2911217
prob -117.3822784
prob -221.8315654
prob -192.3377526
prob -273.3878613
prob -158.7311596
prob -133.7594559
prob -235.0227301
prob -271.2361822
prob -300.5898985
prob -133.9742199
prob -201.2235932
prob -214.9363729
prob -213.129396
prob -149.9786053
prob -174.1397352
prob -291.6731486
prob -175.5877542
prob -151.5253321
prob -258.1761799
prob -130.9300319
prob -274.8197083
prob -362.1171638
prob -238.8415997
prob -263.9148188
prob -176.0725354
prob -301.0714183
prob -154.814648
prob -313.6807703
prob -96.75389902
prob -175.3889275
prob -171.3838374
prob -160.7915887
prob -295.1976066
prob -278.3435151
prob -351.7565316
prob -282.3298427
prob -123.6376823
prob -328.9640835
prob -361.9792491
prob -145.2586754
prob -221.1768138
prob -294.9654105
prob -324.8152967
prob -177.6376837
prob -248.8282921
prob -283.1410968
prob -247.1345143
prob -299.6004285
prob -342.8243443
prob -257.2680604
prob -226.0822195
prob -188.7406777
prob -218.3701598
prob -194.0480634
prob -293.6923531
prob -189.7336054
prob -180.9413126
prob -276.7019229
prob -199.1929523
prob -215.6126339
prob -218.4468615
prob -217.0841132
prob -246.791986
prob -282.4893495
prob -216.992332
prob -190.5704452
prob -308.0079153
prob -258.1242271
prob -282.4829758
prob -320.3417628
prob -271.8722743
prob -176.6761997
prob -229.015796
prob -174.8719367
prob -381.3194967
prob -255.4210403
prob -245.9470071
prob -199.1054901
prob -180.1889829
prob -106.8455846
prob -187.3444592
prob -288.8594202
prob -200.110395
prob -241.2751967
prob -349.8097919
prob -281.8587857
prob -235.0052817
prob -192.9414881
prob -117.2371828
prob -219.0745266
prob -253.2552355
prob -121.7640533
prob -172.2160962
prob -327.3962393
prob -408.8431566
prob -270.308534
prob -93.26583312
prob -178.8530276
prob -130.0711363
prob -154.5044337
prob -198.5878884
prob -349.9282188
prob -171.9004812
prob -217.0546169
prob -224.0095958
prob -212.5604054
prob -162.3203548
prob -221.9523925
prob -233.7969951
prob -350.5649528
prob -231.4750453
prob -181.5080751
prob -200.8190201
prob -243.839777
prob -248.6383675
prob -98.96597213
prob -275.615833
prob -328.5033593
prob -205.9690316
prob -331.3975969
prob -238.882848
prob -209.7778453
prob -258.7910746
prob -344.7135471
prob -223.7969278
prob -285.4249333
prob -94.39372045
prob -220.7117995
prob -318.3128357
prob -302.2876048
prob -110.9955057
prob -355.0484482
prob -283.8876199
prob -173.7907413
prob -162.9072466
prob -191.1484169
prob -268.3412805
prob -313.9838517
prob -172.6158307
prob -353.4369685
prob -251.3825413
prob -238.9342332
prob -290.4519759
prob -231.16311
prob -208.2881073
prob -253.4698639
prob -321.4446738
prob -266.7738268
prob -191.1232785
prob -207.9386866
prob -316.703095
prob -148.7301238
prob -399.0506234
prob -140.1432428
prob -216.6404323
prob -213.8148041
prob -234.795978
prob -305.8363449
prob -216.6551415
prob -293.6362772
prob -146.3346592
prob -295.2933232
prob -199.5333482
prob -201.3407201
prob -170.1859319
prob -130.4650262
prob -189.1977831
prob -291.8193104
prob -296.0510543
prob -146.4427358
prob -280.6341088
prob -242.5343022
prob -315.1560068
prob -227.7521532
prob -397.7362793
prob -127.6057807
prob -316.8201801
prob -203.0812743
prob -317.1197835
prob -200.2876005
prob -251.2997555
prob -254.6230194
prob -245.3738658
prob -411.5691206
prob -314.3840539
prob -175.174248
prob -356.2344952
prob -214.7998709
prob -255.526527
prob -330.4171398
prob -178.1799858
prob -297.8902991
prob -307.5622249
prob -51.05354339

 new best -19471.30367 -19486.42318 -533.1309687 5.709538201 2336.981235 5.3136388 -51.05354339
prob -133.2862492
prob -135.9266368
prob -224.6443265
prob -294.1650912
prob -213.4975672
prob -352.4491387
prob -305.3223214
prob -247.6791191
prob -310.0257362
prob -146.9194039
prob -217.7117091
prob -199.1172484
prob -188.7865273
prob -260.7864205
prob -269.9546325
prob -304.1992072
prob -281.6850644
prob -232.6365075
prob -129.6439103
prob -224.4488186
prob -310.0946489
prob -212.4754683
prob -346.9876913
prob -370.3959249
prob -161.7532621
prob -132.1409966
prob -314.5803795
prob -206.2574268
prob -198.7097588
prob -187.0986016
prob -353.3987543
prob -225.2620854
prob -314.6418755
prob -263.7341566
prob -276.8297397
prob -172.3466997
prob -354.1647972
prob -313.0338955
prob -111.4586682
prob -232.3933679
prob -182.0888783
prob -213.4146793
prob -329.5566331
prob -258.3877137
prob -235.3492278
prob -143.3764465
prob -193.9755487
prob -299.8933784
prob -189.2416201
prob -268.6810726
prob -272.4701508
prob -339.7279598
prob -224.1818766
prob -169.6349079
prob -202.4318271
prob -221.3066688
prob -411.4816228
prob -270.9049049
prob -149.4221021
prob -341.1458727
prob -295.6757767
prob -297.0970938
prob -169.0352324
prob -242.9816137
prob -160.1266281
prob -226.8633898
prob -242.7086695
prob -136.3130238
prob -282.1740537
prob -255.3989543
prob -203.0395678
prob -238.8234367
prob -200.3094262
prob -212.4903255
prob -291.9315379
prob -171.9645854
prob -214.2329061
prob -155.631052
prob -390.7624753
prob -260.5282452
prob -295.3793626
prob -233.2360122
prob -166.0076352
prob -109.881565
prob -380.5203544
prob -146.625814
prob -160.644508
prob -317.0917412
prob -207.3458236
prob -314.9005918
prob -104.522227
prob -196.6349463
prob -315.7656918
prob -348.5789881
prob -268.9587779
prob -155.7420913
prob -149.4527561
prob -264.7724008
prob -332.5213889
prob -105.7343434
prob -126.2049354
prob -223.7240414
prob -427.5942383
prob -294.0628764
prob -339.0185859
prob -182.167543
prob -179.2883847
prob -177.4015712
prob -226.0941193
prob -258.1520826
prob -362.3939916
prob -204.7407888
prob -343.7688241
prob -238.8220652
prob -318.3636007
prob -277.448238
prob -315.3785876
prob -249.1950516
prob -263.5399094
prob -321.4498755
prob -292.5935597
prob -344.1253041
prob -248.6913177
prob -251.180434
prob -232.3505887
prob -224.0265667
prob -224.7765421
prob -217.7448733
prob -311.0770352
prob -280.6685416
prob -140.6390125
prob -251.3298291
prob -148.1856427
prob -303.72648
prob -253.0605315
prob -265.3994822
prob -293.443291
prob -198.1683038
prob -175.3987909
prob -130.3689676
prob -208.3725497
prob -295.3924638
prob -317.3148809
prob -387.2269851
prob -281.1429957
prob -329.7308863
prob -157.3567375
prob -333.2518331
prob -336.6088941
prob -215.9783568
prob -157.0810661
prob -280.4750082
prob -299.1969525
prob -175.3315598
prob -337.4251493
prob -192.9160148
prob -183.7892987
prob -107.924203
prob -101.735397
prob -179.2374022
prob -261.7709713
prob -358.389242
prob -264.1632039
prob -184.0359236
prob -384.1869117
prob -224.7086794
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative -14.1042733 analytic -14.1042733
acceptance 0.682375
timesteps   avg   err
0.006083717324  -19471.39809  0.08062317437
0.05531220344  -19486.44247  0.1224563277
1.098411316  -542.0193142  132.835659
0.1234302653  5.293975567  0.9091978918
1.0541956  2402.823107  61.64798719
0.001221708572  5.027942192  0.2754963144

c[0] = -19471.39809 +/- 0.08062317437
c[1] = -19486.44247 +/- 0.1224563277
c[2] = -542.0193142 +/- 132.835659
c[3] = 5.293975567 +/- 0.9091978918
c[4] = 2402.823107 +/- 61.64798719
c[5] = 5.027942192 +/- 0.2754963144
Assessment of the fit:
fitted value -19469.2475 data -19469.06162 +/- 0.2386866021
fitted value -19472.67538 data -19472.91209 +/- 0.1824016432
fitted value -19473.50155 data -19473.40941 +/- 0.1821665035
fitted value -19473.24116 data -19473.66228 +/- 0.1540107974
fitted value -19472.95112 data -19473.23056 +/- 0.1770901115
fitted value -19472.19999 data -19471.3645 +/- 0.2461646212
fitted value -19473.7674 data -19473.74408 +/- 0.1834546702
fitted value -19473.24858 data -19473.50973 +/- 0.181134859
fitted value -19472.86356 data -19473.06261 +/- 0.1576947666
fitted value -19472.35123 data -19472.30654 +/- 0.1572564139
fitted value -19472.17912 data -19471.42344 +/- 0.1274873928
fitted value -19471.81468 data -19472.26098 +/- 0.1614681087
fitted value -19471.67858 data -19471.36149 +/- 0.1711038626
fitted value -19471.53866 data -19471.80548 +/- 0.1436593298
fitted value -19487.9275 data -19487.34243 +/- 0.2215584649
fitted value -19486.5614 data -19486.79659 +/- 0.1401297033
minimum skew nan
